packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -9,6 +9,22 @@ id: User feedback
 source: design-guidelines
 ---
 
-Design guidelines
+Note: User feedback lives in its own package at [@patternfly/react-user-feedback](https://github.com/patternfly/react-user-feedback).
 
-COMING SOON!
+**User feedback** is a modal component that on-premises products can use to collect asynchronous feedback from users. 
+
+## Behavior
+
+When installed, user feedback may be accessed in a couple of ways: 
+
+By selecting a masthead’s help icon and choosing “Share feedback” in the dropdown menu:
+
+![Image of expanded dropdown menu with link to sharing feedback](/packages/module/patternfly-docs/content/img/dropdown-selection.png)
```

Comment:
The documentation framework can't find these images dropdown-selection.png and wizard-selection.png.  If you move the img folder under design-guidelines, you can access the images.  You will have to change this to ./img/dropdown-selection.png.